src/extractor/scripting_environment_lua.cpp

```diff
@@ -34,7 +34,12 @@ namespace
 template <class T>
 auto get_value_by_key(T const &object, const char *key) -> decltype(object.get_value_by_key(key))
 {
-    return object.get_value_by_key(key, "");
+    auto v = object.get_value_by_key(key);
```

Comment:
Yeah `{` goes on the next line and we try not use short names like `v` but rather `value`.